### ISSUE TITLE
encoder and decoder

### ISSUE THREAD

**Issue by qihan-zhang-zach** [brambozz/medical-diffusion-on-a-budget#4]
may i ask if there is an extra encoder and decoder in that latent diffusion model block? 
i thought the process is like

img -> encoder -> img_encoded
img_encoded -> diffusion model -> img_encoded_diffused
img_encoded_diffused -> decoder -> img_diffused (img_generated)

but i cannot find the encoder or decoder in your codes.

**Comment by brambozz** [comment]
Yes that's correct, but that is all included in the base stable diffusion checkpoint. I do not implement any of the diffusion code in my paper, and instead use [this Web UI](https://github.com/AUTOMATIC1111/stable-diffusion-webui) for training the embeddings and generating images.
